### PR TITLE
30 - Implement About Page

### PR DESCRIPTION
**Pull request by nozomione**
## Issue Number
#30 

Figma Mockup: [About](https://www.figma.com/file/2cxEjUKA4ACTjDYSnDEWpR/refinebio-master?node-id=187%3A6298&t=rLN6bTbL0ji4Rs2d-0)

(This PR is a dependent of the PR [#29](https://github.com/AlexsLemonade/refinebio-web/pull/32) and was created based on the branch `nozomione/29-implement-landing-page`)

Implemented the about page using the global components.

<details>
<summary>Files related to this PR</summary>

---
</details>

- [x] UI review w/ @dvenprasad - please review the UI [here](https://refinebio-web-git-nozomione-30-implement-about-page-ccdl.vercel.app/about), thank you!

(After passing the UI review, @davidsmejia will be added to the reviewers.)

## Types of changes

What types of changes does your code introduce?

- New feature (non-breaking change which adds functionality)

## Checklist

- [x] Lint and unit tests pass locally with my changes

### CODE REVIEW

**Review by dvenprasad — CHANGES_REQUESTED** [review]
All of the components, font sizes looks good! I have layout comment

There's two margins, the outer one which is the bounds of the navigation bar and an inner one which is for the content: 
Screen Shot 2022-11-16 at 10 26 28 AM

Here, there seems to be only one which is the width of the navigation bar, but the content should be contatined within the bounds of the second one like above:
Screen Shot 2022-11-16 at 10 27 00 AM

**Comment by nozomione** [comment]
> All of the components, font sizes looks good! I have layout comment
> 
> There's two margins, the outer one which is the bounds of the navigation bar and an inner one which is for the content: Screen Shot 2022-11-16 at 10 26 28 AM
> 

In this case, the page content will be shifted more to the right side. Or do you prefer it to be centered of the page? Please let me know!

**Comment by dvenprasad** [comment]
Let's center the content

**Comment by nozomione** [comment]
> Let's center the content

@dvenprasad, the UI is ready for your review, thank you!

**Review by dvenprasad — APPROVED** [review]
Looking good! 🦔 🦔 🦔

File: src/components/About/AboutHero.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      <Text size={setResponsive('medium', 'large')} textAlign="center">
+        {count} {text}
+      </Text>
+    </Box>
+  )
+}
+
+const HeroBody = () => {
+  const { setResponsive } = useResponsive()
+  // TEMP
```
[the remark this answers is not in the document]
Comment (nozomione) [reply src/components/About/AboutHero.js:45]: The current refinebio pull those data from the [apiData.json](https://github.com/AlexsLemonade/refinebio-frontend/blob/dev/src/apiData.json) file (a cache stored in the `src` directory). Once the file is added(if we'll be using it), those temp object will be removed 🗑

File: src/themes/colors.js [hunk 2]
```diff
@@ -84,6 +84,15 @@ export const colors = {
   gradient:
     'linear-gradient(180deg, rgba(12, 52, 131, 0.937981) 0%, #6B8CCE 131.35%, #A2B6DF 190.36%, #A1B6DF 190.36%, #A2B6DF 190.36%)',
   'gradient-light': 'linear-gradient(180deg, #DFE9F3 7.13%, #FFFFFF 75%)',
+  gradient_light: 'linear-gradient(360deg, #DFE9F3 7.13%, #FFFFFF 75%)',
```
[the remark this answers is not in the document]
Comment (nozomione) [reply src/themes/colors.js:87]: Please see the PR [107- Update All Themeing Variable Names To Camel Case](https://github.com/AlexsLemonade/refinebio-web/pull/110). Btw, thank you for coming up with this idea 💡 to reduce the merge conflict!

**Pull request merged**